testsuites/CBLTester/upgrade_tests/test_cbl_upgrade.py

```diff
@@ -258,4 +204,141 @@ def test_upgrade_cbl(params_from_base_suite_setup):
     else:
         db_path = '/'.join(db_path.split('/')[:-1])
     if db.exists(upgrade_cbl_db_name, db_path):
+        log_info("Delete DB - {}".format(upgrade_cbl_db_name))
         db.deleteDB(cbl_db)
+
+
+@pytest.mark.listener
+@pytest.mark.upgrade_test
+def test_queries_on_upgrade_cbl(params_from_base_suite_setup):
+    """
+    @summary:
+    1. Upgrade the db from lower CBl version
```

Comment:
can you mention explicitely that encrypted db is copied for 2.1.3 and up and unencrypted for versions below 2.1.3

Reply:
encrypted prebuilt databases is copied for 2.1.0 and up and unencrypted database below 2.1.0

Reply:
added